Jenkinsfile

```diff
@@ -592,65 +592,14 @@ pipeline {
                         buildHipClangJobAndReboot()
                     }
                 }
-            }
-        }
-        stage("Smoke MLIR") {
-            when {
-                expression { params.BUILD_SMOKE_MLIR }
-            }
-            parallel{
-                stage('Fp32 Hip MLIR') {
-                    when {
-                        beforeAgent true
-                        expression { (params.TARGET_VEGA20 || params.TARGET_VEGA10) && params.DATATYPE_FP32 }
-                    }
-                    agent{ label rocmnode("vega") }
-                    steps{
-                        buildHipClangJobAndReboot(setup_flags: MLIR_flags, build_env: extra_log_env, config_targets: Smoke_targets, test_flags: ' --verbose ', mlir_build: "ON")
-                    }
-                }
-                stage('Fp16 Hip MLIR') {
-                    when {
-                        beforeAgent true
-                        expression { (params.TARGET_VEGA20 || params.TARGET_VEGA10) && params.DATATYPE_FP16 }
-                    }
-                    agent{ label rocmnode("vega") }
-                    steps{
-                        buildHipClangJobAndReboot(setup_flags: MLIR_flags + Fp16_flags, build_env: extra_log_env, config_targets: Smoke_targets, test_flags: ' --verbose ', mlir_build: "ON")
-                    }
-                }
-                stage('Fp16 Hip MLIR gfx908') {
+                stage('Fp16 Hip gfx908') {
```

Comment:
It looks like this PR adds 'Fp16 Hip gfx908' to `stage("Smoke Aux 1")`

Reply:
Before you raise this question, I was just looking at the `Smoke Fp32` and `Smoke Aux1` stages before the original MLIR stage. Now I just realized that there are more specific fp16 stages in the `Smoke Fp16/Bf16/Int8`.

Based on this observation I will remove 'Fp16 Hip gfx908`